src/utils/transaction.ts

```diff
@@ -230,7 +248,7 @@ const transaction = {
     const nowTs = Math.floor(Date.now() / 1000);
     const nowHeight = await storage.getCurrentHeight();
     const rewardLock = storage.version?.reward_spend_min_blocks;
-    const isHeightLocked = (!(rewardLock && tx.height)) ? false : ((tx.height + rewardLock) < nowHeight);
```

Comment:
This was a bug before, right? It's the opposite comparison

Reply:
Yes, but the only place it is used is the wallet facade's `getTxBalance` which returns the sum of locked and unlocked balances, so this error was being overlooked.